static/defaults.config.php

```diff
@@ -470,6 +470,10 @@
 		// Whether to use Cache to store session data or to use PHP native session storage.
 		'session_handler' => 'database',
 
+		// remote_avatar_lookup (Boolean)
+		// Perform an avatar lookup via the activated services for remote contacts
+		'remote_avatar_lookup' => false,
```

Comment:
How is the "activated service" chosen if there's more than one addon enabled providing a default avatar?

Reply:
The addons are called alphabetically if I remember correctly. So you should only activate a single one. (But that's no new situation, that was always the case when fetching avatars for the own users)